tests/templates/kuttl/mount-dags-gitsync/31-assert.yaml.j2

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+metadata:
+  name: metrics
```

Comment:
Apparently this fields are [bogus](https://kuttl.dev/docs/testing/reference.html#teststep) (they are ignored by `kuttl`):

```suggestion
```